webapps/api/backend/views/vdisks.py

```diff
@@ -101,6 +109,27 @@ def set_config_params(self, vdisk, new_config_params, version):
         return VDiskController.set_config_params.delay(vdisk_guid=vdisk.guid, new_config_params=new_config_params)
 
     @link()
+    @log()
+    @required_roles(['read'])
+    @return_list(VDisk)
+    @load(VDisk)
+    def get_children(self, vdisk, hints):
```

Comment:
Shouldn't this function be recursive?  Because children can have children and so on right??